devsetup/scripts/standalone.sh

```diff
@@ -56,6 +56,12 @@ sudo dnf update -y
 EOF
 fi
 
+if [[ -e /run/systemd/resolve/resolv.conf ]]; then
+    HOST_PRIMARY_RESOLV_CONF_ENTRY=$(cat /run/systemd/resolve/resolv.conf | grep ^nameserver | grep -v '192.168' | head -n1 | cut -d' ' -f2)
+else
+    HOST_PRIMARY_RESOLV_CONF_ENTRY=$(cat /etc/resolv.conf | grep ^nameserver | grep -v '192.168' | head -n1 | cut -d' ' -f2)
+fi
+
```

Comment:
@cescgina Can you please try to check if this will yield a good value in a CI host? The variable should contain a DNS IP that the host is configured with to resolve names in the outside world (as opposed to resolve names e.g. from CRC).

Reply:
@jistr tried the second command on a CI host, it retruns `127.0.0.1`, and `/run/systemd/resolve/resolv.conf` does not exist there